src/_gettsim/social_insurance_contributions/ges_pflegev.py

```diff
@@ -29,14 +29,71 @@ def ges_pflegev_zusatz_kinderlos(
     return out
 
 
+def ges_pflegev_beitr_regulär_besch_m(
+    _ges_krankenv_bruttolohn_m: float,
+    anz_kinder_u_25: int,
+    ges_pflegev_zusatz_kinderlos: bool,
+    sozialv_beitr_params: dict,
+) -> float:
+    """Calculate care insurance contributions for regular jobs.
+
+    Add additional contribution for childless individuals and substract contribution
+    for individuals with two or more children under 25.
+
+    Parameters
+    ----------
+    _ges_krankenv_bruttolohn_m
+        See :func:`_ges_krankenv_bruttolohn_m`.
+    anz_kinder_u_25
+        See basic input variable :ref:`anz_kinder_u_25 <anz_kinder_u_25>`.
+    ges_pflegev_zusatz_kinderlos
+        See :func:`ges_pflegev_zusatz_kinderlos`.
+    sozialv_beitr_params
+        See params documentation :ref:`sozialv_beitr_params <sozialv_beitr_params>`.
+
+    Returns
+    -------
+    Monthly care insurance contributions for regular jobs.
+
+
+    """
+
+    beitr_regulär_beschäftigt_m = (
+        _ges_krankenv_bruttolohn_m
+        * sozialv_beitr_params["beitr_satz"]["ges_pflegev"]["standard"]
+    )
+
+    # Add additional contribution for childless individuals
+    if ges_pflegev_zusatz_kinderlos:
+        beitr_regulär_beschäftigt_m += (
+            _ges_krankenv_bruttolohn_m
+            * sozialv_beitr_params["beitr_satz"]["ges_pflegev"]["zusatz_kinderlos"]
+        )
+
+    # Substract contribution for individuals with two or more children under 25
+    for i in range(2, 5):
+        if anz_kinder_u_25 == i:
```

Comment:
I think we can get ride fo the loop and the second part of the code by something like:

```
if anz_kinder_u_25 >= 2:
    .... * min(anz_kinder_u_25 - 1, 4)